src/markdown-pages/sheet-pan-chicken-thighs.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: "/recipes/sheet-pan-chicken-thighs"
+date: "2021-12-12"
+title: "Sheet pan chicken thighs"
+mealType: "dinner"
+imageUrl: "https://recipes-md.s3.amazonaws.com/images/no-image-placeholder.svg"
+---
+
+## About the recipe
+
+Serves 4
+
+Total time: 45 mins
+
+Source: [littlesunnykitchen.com](https://littlesunnykitchen.com/sheet-pan-chicken-thighs/)
+
+## Goes well with
+
+dinner rolls, rice pilaf, brown rice, roasted potatoes
+
+## Ingredients
+
+- 2 pounds chicken thighs
+- 3 tablespoons olive oil
+- 1 tablespoon Italian seasoning
+- 1 teaspoon salt
+- 1/4 teaspoon ground black pepper
+- 2 tablespoons fresh lemon juice
+- 1 tablespoon Dijon mustard
+- 10 ounces brussels sprouts
+- 1 red onion, sliced into 8 wedges
+- 6 cloves garlic
+
+## Steps
+
+To marinate the chicken, put the chicken thighs to a large bowl. Then add olive oil, Italian seasoning, salt, pepper, lemon juice, and dijon mustard and mix to coat. Let sit while preparing the other ingredients.
+
+Place trimmed and halved brussels sprouts on a sheet pan, followed by red onion, and garlic cloves.
+
+Drizzle with olive oil, and season with salt and pepper. Toss so that everything is well coated.
+
+Place marinated chicken thighs skin side up on the sheet pan. Bake uncovered at 400ºF. If bone-in for 30 minutes, otherwise, 25 minutes. Then crisp the skin up and brown it under the broiler for a few minutes.
```

Comment:
```suggestion
Place marinated chicken thighs skin side up on the sheet pan. Bake uncovered at 400ºF for 30 minutes if bone-in, 25 minutes otherwise. Then crisp the skin up and brown it under the broiler for a few minutes.
```